sematic/db/models/resolution.py

```diff
@@ -211,6 +212,13 @@ class Resolution(HasUserMixin, Base, JSONEncodableMixin):
     settings_env_vars: Dict[str, str] = Column(
         types.JSON(), nullable=False, default=lambda: {}, info={REDACTED_KEY: True}
     )
+
+    # WARNING: deprecated column, do not remove until Python-packaged SQLite supports
```

Comment:
these columns are being restored in the model until we have broad support for dropping columns in sqlite.  a few more instances of this in the rest of the pr.

Reply:
Please add this formulation, so it can be found by grepping, like the other places. Please propagate everywhere. 
```suggestion
    # TODO #303: WARNING: deprecated column, do not remove until Python-packaged SQLite supports
```